perun-db/addGroupMembershipManagerPolicies

```diff
@@ -0,0 +1,112 @@
+my @readAttributes = (
+	"urn:perun:vo:attribute-def:def:blockManualMemberAdding",
+	"urn:perun:group:attribute-def:def:groupMembershipExpirationRules",
+	"urn:perun:group:attribute-def:def:blockManualMemberAdding"
+);
+my @writeAttributes = (
+	"urn:perun:member_group:attribute-def:def:groupMembershipExpiration"
+);
+my ($user, $pwd, $overwrite);
+
+GetOptions ("help|h" => sub { print help(); exit 0;},
+	"user|u=s" => \$user,
+	"password|w=s" => \$pwd,
+	"overwrite|o" => \$overwrite) || die help();
+
+if (!defined $user) { print "[ERROR] Username for Perun DB is required! Use --help | -h to print help.\n"; exit 1; }
+if (!defined $pwd) { print "[ERROR] Password for Perun DB is required! Use --help | -h to print help.\n"; exit 1; }
+
+my $dbh = DBI->connect('dbi:Pg:dbname=perun',$user,$pwd,{RaiseError=>1,AutoCommit=>0,pg_enable_utf8=>1}) or die EPERM," Connect";
+
+my $sthCollections = $dbh->prepare(q{select id, attr_name from attr_names;});
+
+my $sthNewCollection = $dbh->prepare(q{insert into attribute_policy_collections (id, attr_id, action) values (?,?,?);});
+
+my $sthNewPolicy = $dbh->prepare(q{insert into attribute_policies (id, role_id, object, policy_collection_id) values (?,?,?,?);});
+
+my $sthInsertRole = $dbh->prepare(q{insert into roles (id, name) values (?,'groupmembershipmanager');});
+
+my $sthDelCollections = $dbh->prepare(q{delete from attribute_policy_collections where id=?;});
+
+my $sthExistingPolicies = $dbh -> prepare(q{select policy_collection_id from attribute_policies where role_id=?;});
+
+#-------------------------------------------------------------------------------------------------- execution
+
+# Insert GROUPMEMBERSHIPMANAGER role to DB if not created yet
+my $roleId = $dbh->selectrow_array(q{select id from roles where name='groupmembershipmanager'});
+unless ($roleId) {
+	$roleId = $dbh->selectrow_array('select nextval(\'roles_id_seq\')');
+	$sthInsertRole->execute($roleId);
+	$dbh->commit() or die $dbh->errstr;
+	print "INFO: Groupmembershipmanager role created\n";
+}
+
+# Remove all collections where any policy with Groupmembershipmanager exists
+if ($overwrite) {
+	$sthExistingPolicies->execute($roleId);
+	while (my @row = $sthExistingPolicies->fetchrow_array()) {
+		my ($collectionId) = @row;
+		$sthDelCollections->execute($collectionId);
+	}
+	$dbh->commit() or die $dbh->errstr;
+	print "INFO: Existing policy collections successfully removed.\n";
+}
+
+# Fetch attribute definitions
+$sthCollections->execute();
+while (my @row = $sthCollections->fetchrow_array()) {
+	my ($attrId, $attrName) = @row;
+	my $attrType = (split ':', $attrName)[2];
+
+	# add READ policy for GROUPMEMBERSHIPMANAGER
+	my ($shouldRead) = grep {$attrName eq $_} @readAttributes;
+	my $object = ($attrType eq "member" || $attrType eq "vo" || $attrType eq "user") ? "Vo" : "Group";
+
+	if ($attrType eq "member" || $attrType eq "member_group" || $attrType eq "user_facility" || $attrType eq "user" || $shouldRead) {
```

Comment:
If it is not too much work, I would add a check that GROUP ADMIN has a READ policy for the attribute.

Reply:
extended the select query for attribute definitions to ensure that